<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="left_btn">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>360</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down_btn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>390</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>370</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>330</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up_btn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>330</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>/\</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_btn">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>360</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sputnik">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Спутник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scheme">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Схема</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hybrid">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Гибрид</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="find_address">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>120</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти адрес</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>170</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сброс поискового 
 адресa</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="index_chb">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Индекс</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>435</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
